Formula/diffuse.rb

```diff
@@ -0,0 +1,29 @@
+class Diffuse < Formula
+  desc "Graphical tool for merging and comparing text files"
+  homepage "https://mightycreak.github.io/diffuse/"
+  url "https://github.com/MightyCreak/diffuse/archive/refs/tags/v0.7.3.tar.gz"
+  sha256 "cceff03beabc83fc76c49ecdad6743b7bd709063167af851fe2ebae7a10b1394"
+  license "GPL-2.0-or-later"
+
+  depends_on "gettext" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+
+  depends_on "gtk+3"
```

Comment:
When I studied Diffuse on macOS, I found that we actually need a few other deps, see:
[brew install meson python3 py3cairo pygobject3 gtk+3 librsvg](https://github.com/MightyCreak/diffuse/blame/master/docs/developers/developers-setup.md#L45)